src/Engine/MainWindow.cpp

```diff
@@ -107,6 +106,22 @@ GLFWwindow* MainWindow::GetGLFWWindow() const {
     return window;
 }
 
+void MainWindow::SetWindowMode(bool fullscreen, bool borderless) const {
+    glfwSetWindowAttrib(window, GLFW_DECORATED, borderless ? GLFW_FALSE : GLFW_TRUE);
+
+    int x, y, width, height;
+    glfwGetWindowPos(window, &x, &y);
+    glfwGetWindowSize(window, &width, &height);
+    glfwSetWindowMonitor(window, fullscreen ? glfwGetPrimaryMonitor() : nullptr, 50, 50, width, height, GLFW_DONT_CARE);
+        
+}
+
+void MainWindow::GetWindowMode(bool& fullscreen, bool& borderless) const {
+    GLFWmonitor* monitor = glfwGetWindowMonitor(window);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'monitor' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)